packages/serde/src/player.ts

```diff
@@ -26,5 +26,6 @@ export function decodePlayer(rawPlayer: RawPlayer): Player {
     claimedShips: rawPlayer.claimedShips,
     finalRank: rawPlayer.finalRank.toNumber(),
     claimedReward: rawPlayer.claimedReward,
+    silver: silver,
```

Comment:
Silver is a synthetic field on the `Player` type in client, maybe we should change that